hamocc/aufw_bgc.F90

```diff
@@ -106,30 +106,33 @@ SUBROUTINE AUFW_BGC(kpie,kpje,kpke,ntr,ntrbgc,itrbgc,trc,              &
       use mo_intfcblom,   only: sedlay2,powtra2,burial2,atm2
       use mod_xc,         only: nbdy,itdm,jtdm,mnproc,iqr,jqr,xchalt
       use mod_dia,        only: iotype
-#ifdef AGG
+      ! AGG
       use mo_param1_bgc,  only: iadust, inos
-#endif
-#ifdef BOXATM
+
+      ! BOXATM
       use mo_param1_bgc,  only: iatmco2,iatmn2,iatmo2
-#endif
-#ifdef BROMO
+
+      ! BROMO
       use mo_param1_bgc,  only: ibromo
-#endif
-#ifdef CFC
+
+      ! CFC
       use mo_param1_bgc,  only: icfc11,icfc12,isf6
-#endif
-#ifdef cisonew
-      use mo_param1_bgc,  only: icalc13,icalc14,idet13,idet14,idoc13,idoc14,iphy13,iphy14,isco213,isco214,izoo13,izoo14
-#endif
-#ifdef natDIC
+
+      ! cisonew
+      use mo_param1_bgc,  only: icalc13,icalc14,idet13,idet14,idoc13,idoc14,iphy13,iphy14,isco213,isco214,izoo13,izoo14, &
+                              & issso13,issso14,isssc13,isssc14,ipowc13,ipowc14,iatmc13,iatmc14,                         &
+                              & iatmnco2,issso13,issso14,isssc13,isssc14,ipowc13,ipowc14,iatmc13,iatmc14
+      use mo_control_bgc, only: rmasks
+
+      ! natDIC
       use mo_param1_bgc,  only: inatalkali,inatcalc,inatsco212
       use mo_carbch,      only: nathi
-#endif
-#ifndef sedbypass
+
+      ! NOT sedbypass
       use mo_param1_bgc,  only: ipowaal,ipowaic,ipowaox,ipowaph,ipowasi,ipown2,ipowno3,isssc12,issso12,issssil,issster
-#endif
 
- 
+      use mo_ifdefs
```

Comment:
Here and potentially in other routines: it would be nice to explicitly incorporate all used switches, coming from `mo_ifdefs` (or potentially later from `mo_control_bgc`).

Reply:
In the latest changes I have pushed back there is no long the `mo_ifdefs.F90 `routine - all the logic has been migrated to `mo_control_bgc.F90`.

Reply:
Done.